easybuild/easyconfigs/c/CrystFEL/CrystFEL-0.11.1-foss-2024a.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'MesonNinja'
+
+name = 'CrystFEL'
+version = '0.11.1'
+
+homepage = 'http://www.desy.de/~twhite/crystfel/'
+description = """CrystFEL is a suite of programs for processing diffraction data acquired "serially" in a "snapshot"
+manner, such as when using the technique of Serial Femtosecond Crystallography (SFX) with a free-electron laser
+source."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+source_urls = ['http://www.desy.de/~twhite/crystfel/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['bd9b8df5d627a30c920bf397d04641d266e1a013b24623967d7f5deac97594a4']
+
+builddependencies = [
+    ('Meson', '1.4.0'),
+    ('Ninja', '1.12.1'),
+    ('pkgconf', '2.2.0'),
```

Comment:
flex seems to be required.
../crystfel-0.11.1/libcrystfel/meson.build:96:7: ERROR: Program 'flex' not found or not executable